automol/vmat.py

```diff
@@ -163,15 +167,13 @@ def atom_indices(vma, symb, match=True):
     return idxs
 
 
-def coordinate_key_matrix(vma, shift=0):
+def coordinate_key_matrix(vma: VMatrix, shift: int = 0) -> key_matrix:
```

Comment:
The return type here is a little weird. Each value in the matrix is a tuple describing the coordinate (e.g. (2, 1) for a bond distance coordinate, (3, 2, 1) for an angle coordinate, etc.). Let's define the following above with the other types:
```
CoordinateKey = tuple[int, ...] | None
CoordinateKeyRow = tuple[CoordinateKey, CoordinateKey, CoordinateKey]
CoordinateKeyMatrix = tuple[CoordinateKeyRow, ...]
```
and then put the return type as `-> CoordinateKeyMatrix:`